### ISSUE TITLE
Make karma plugins and options configurable

### ISSUE THREAD

**Issue by anthonymayer** [Sanjo/meteor-jasmine#116]
I'm interested in using both https://github.com/karma-runner/karma-sauce-launcher and https://github.com/karma-runner/karma-firefox-launcher. It probably wouldn't hurt to hardcode these in as options, but it seems like an application should be able to specify a custom karma plugin and configuration options. The configuration options are definitely necessary for the sauce runner since with that plugin you can then specify which browsers to actually run and it doesn't seem sane to limit that inside of the meteor-jasmine code. Since this is more information than it makes sense to have in env variables, how would you feel about a top level config file in the application (like meteor_jasmine_config.js or something) that sets the properties to KARMA_START_OPTIONS (or something like that)?

What I have no good answer for would be how to actually require the npm package for the karma runner. Does it make sense to have people wrap the runners in meteor packages? It looks like you started down that path with https://github.com/Sanjo/meteor-karma-phantomjs-launcher-nonet, but didn't actually use it. Would it work though? I definitely like the idea of people being able to include the runner they need as a meteor package.

I'm happy to work on this, just figured I'd get some guidance on the approach first.

**Comment by Sanjo** [comment]
I think I can add the launchers as dependency of sanjo:karma.

I can add an API in sanjo:jasmine to configure additional Karma options. You could use it like:

```javascript
if (Package['sanjo:jasmine']) {
  Jasmine.setKarmaConfig({
    sauceLabs: {
        testName: 'Web App Unit Tests'
    },
    customLaunchers: customLaunchers,
    browsers: Object.keys(customLaunchers),
  });
}
```

__Tasks for implementation:__

* [ ] Add sauce labs and firefox launcher to sanjo:karma.
* [ ] Add API `Jasmine.setKarmaConfig`
* [ ] Karma must be started after the app had the change to call the setKarmaConfig API.
* [ ] Blacklist the config options that should not be changed by the user.

**Comment by Sanjo** [comment]
Firefox is available since 0.11.0. The config API for Sauce Labs must still be done.
